# Basic Params
TriggerTopic:               '/move_base_simple/goal'
OdomTopic:                  '/drone_0/odom_payload'
TrajectoryTopic:            '/trajectory_generator_node/trajectory'
OdomFrame:                  'world'
BoxMin:                     [-5.0, -3.0, 0.2]
BoxMax:                     [ 5.0,  3.0, 1.6]
Round:                      1

# Advanced Params
DiagWidth:                  1.0
SideWidth:                  1.0

MaxAccRate:                  9.50
MaxVelRate:                  5.0
WeightT:                     4096.0

ChiVec:                      [1024.0, 1024.0, 1024.0]
RelCostTol:                  1.0e-4
C2Diffeo:                    True

sample_n: 2
ballsize: 0.15
quadsize: 0.25
l_length: 0.58